WcaOnRails/spec/controllers/results_submission_controller_spec.rb

```diff
@@ -0,0 +1,84 @@
+# frozen_string_literal: true
```

Comment:
Thanks for writing the specs! Controller specs are deprecated (starting from Rails 5) though, see [this PR description](https://github.com/thewca/worldcubeassociation.org/pull/1312#issue-206868269). Please, make it a request spec instead.

Reply:
Thanks for changing this to a request spec! Can you rename it appropriately, though? Currently the filename is `.../controllers/results_submission_controller_spec.rb`, and it should really be something like `.../requests/results_submission_request_spec.rb`.

Reply:
I'm moving the file to the `requests` subfolder and renaming it `results_submission_spec` (the other files don't have `request` in their name)